reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -47,25 +53,37 @@ private Sinks() { }
 	 */
 	@SuppressWarnings("deprecation")
 	public static <T> Sink<T> multicast() {
-		return new ProcessorSink<>(EmitterProcessor.create(Queues.SMALL_BUFFER_SIZE));
+		EmitterProcessor<T> processor = EmitterProcessor.create(Queues.SMALL_BUFFER_SIZE);
+		return SinksHelper.toSink(processor, processor);
 	}
 
 	/**
 	 * A {@link Sink} with the following characteristics:
 	 * <ul>
 	 *     <li>Multicast</li>
-	 *     <li>Backpressure : this sink honors downstream demand of individual subscribers.</li>
+	 *     <li>Backpressure : this sink is not able to honor downstream demand and will emit `onError` if there is a mismatch.</li>
 	 *     <li>Replaying: No replay. Only forwards to a {@link Subscriber} the elements that have been
-	 *     pushed to the sink AFTER this subscriber was subscribed.</li>
-	 *     <li>Without {@link Subscriber}: Discarding. Pushing elements while there are no {@link Subscriber}
-	 *     registered will simply discard these elements instead of "warming up" the sink.</li>
+	 *     pushed to the sink AFTER this subscriber was subscribed. To the exception of the first
+	 *     subscribe.</li>
+	 * </ul>
+	 */
+	@SuppressWarnings("deprecation")
+	public static <T> Sink<T> multicastNoBackpressure() {
+		DirectProcessor<T> processor = DirectProcessor.create();
+		return SinksHelper.toSink(processor, processor);
+	}
+
+	/**
+	 * A {@link Sink} with the following characteristics:
+	 * <ul>
+	 *     <li>Multicast</li>
+	 *     <li>Backpressure : this sink does not need any demand since it can only signal error or completion</li>
+	 *     <li>Replaying: Replay the terminal signal (error or complete).</li>
 	 * </ul>
-	 * <p>
-	 * <img class="marble" src="doc-files/marbles/sinkNoWarmup.svg" alt="">
 	 */
 	@SuppressWarnings("deprecation")
-	public static <T> Sink<T> multicastNoWarmup() {
-		return new ProcessorSink<>(ReplayProcessor.create(0));
+	public static Sink<Void> coordinator() {
```

Comment:
put it next to `promise`, as both behave closer to `Mono`. That makes me wonder if it would be better backed by a `MonoProcessor`? likely to be used as `toMono()` more often and thus benefit from the 0-overhead conversion? or maybe add a constructor parameter to `DirectProcessor` that allows its `toMono()` method to do `Mono.fromDirect(this)` rather than `next()`?

Reply:
In a separate PR we will have VoidProcessor for this